File: q135/cpp/main.cc
#include <gtest/gtest.h>

#include "Utils.h"
#include "solution.cc"

int main() {
  vector<int> v{4, 1, 0, 2, 3, 5, 6, 7, 8, 9};
  auto result = q135::Solution().candy(v);
  print(result);
  EXPECT_EQ(result, 41);
}

File: q135/java/Main.java
import q135_src.Solution;

public class Main {
    public static void main(String[] args) {
        int[] arr = {4, 1, 0, 2, 3, 5, 6, 7, 8, 9};
        int result = Solution.candy(arr);

        System.out.println("Number of candies need: " + result);
        assert result == 41 : "Test failed";
    }
}
